yarn.lock

```diff
@@ -20260,9 +20334,16 @@ zen-observable@0.8.15:
   resolved "https://registry.yarnpkg.com/zen-observable/-/zen-observable-0.8.15.tgz#96415c512d8e3ffd920afd3889604e30b9eaac15"
   integrity sha512-PQ2PC7R9rslx84ndNBZB/Dkv8V8fZEpk83RLgXtYd0fwUgEjseMn1Dgajh2x6S8QbZAFa9p2qVCEuYZNgve0dQ==
 
+zustand@4.3.3:
+  version "4.3.3"
+  resolved "https://registry.yarnpkg.com/zustand/-/zustand-4.3.3.tgz#c9113499074dde2d6d99c1b5f591e9329572c224"
+  integrity sha512-x2jXq8S0kfLGNwGh87nhRfEc2eZy37tSatpSoSIN+O6HIaBhgQHSONV/F9VNrNcBcKQu/E80K1DeHDYQC/zCrQ==
+  dependencies:
+    use-sync-external-store "1.2.0"
+
 zustand@^4.0.0-rc.0, zustand@^4.0.0-rc.1:
```

Comment:
Looks like these packages are resolved to `4.3.5` and there's another `4.3.3`. Can you try running dedupe to see if the consensus is `4.3.3` here?

Reply:
`4.3.4` is broken FYI, so `4.3.5` is "safe". I will send follow-ups to widget/interface myself later.

Reply:
does not work